src/gsy_e/gsy_e_core/util.py

```diff
@@ -410,15 +410,15 @@ def export_default_settings_to_json_file():
 def area_sells_to_child(trade, area_name, child_names):
```

Comment:
I do not think that this is a good idea, because the FW functions operate on dicts IIRC, while these operate on concrete trade objects. And I think that it makes sense to maintain that way.